packages/datepicker/src/react/calendar-dates.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useContext } from 'react'
+import { css } from 'glamor'
+import { DateObj, RenderProps } from 'dayzed'
+import { HTMLPropsFor } from '@pluralsight/ps-design-system-util'
+import { DateContext } from './context'
+import stylesheet from '../css'
+
+const styles = {
+  calendarDate: (selected: boolean, today: boolean) =>
+    css(
+      stylesheet['.psds-calendar__date'],
+      selected && stylesheet['.psds-calendar__date--selected'],
+      today && stylesheet['.psds-calendar__date--today']
+    ),
+  dateFiller: () => css(stylesheet['.psds-calendar__filler'])
+}
+
+interface ChildrenRenderProps extends HTMLPropsFor<'button'> {}
+
+interface CalendarDatesProps
+  extends HTMLPropsFor<'button'>,
+    Pick<RenderProps, 'getDateProps'> {
+  children: (props: ChildrenRenderProps, dateObj: DateObj) => React.ReactNode
+}
+
+export const CalendarDates: React.FC<CalendarDatesProps> = ({
```

Comment:
Shouldn't we forwardRef? We have tried to make that a default behavior to be relied upon. If not, why not?

Reply:
Nope it's a headless component that uses the render props pattern.